testing/acceptance_test.py

```diff
@@ -186,6 +189,7 @@ def test_crash():
         )
         assert result.ret == 1
 
+    @pytest.mark.xfail(reason="#527: Ignore Python 3.8 failure for the time being")
```

Comment:
Just for reference: I do not think this is py38 specific, but rather pytest related only (https://github.com/pytest-dev/pytest/pull/6981#issuecomment-606496652).
(just leaving this here since I was confused why pytestmaster was not failing anymore)